File: examples/c-sharp-examples/MeshLoadSave.dox.cs
public class MeshLoadSaveExample
{
    public static void Main(string[] args)
    {
        try
        {
            // This example needs an input mesh; create one if you do not have it already
            if (!File.Exists("mesh.stl"))
                MR.MeshSave.toAnySupportedFormat(MR.makeCube(), "mesh.stl");

            // Load mesh
            var mesh = MR.MeshLoad.fromAnySupportedFormat("mesh.stl");

            // Save it in another format
            MR.MeshSave.toAnySupportedFormat(mesh, "mesh.ply");
        }
        catch (Exception e)
        {
            Console.WriteLine("Error: {0}", e.Message);
        }
    }
}

File: examples/js-examples/MeshLoadSave.dox.js
import createMeshLib from '@meshinspector/meshlib';

const ml = await createMeshLib();

// Load mesh
using mesh = ml.MeshLoad.fromAnySupportedFormat('mesh.stl');

// Save mesh (the format is chosen from the file extension)
ml.MeshSave.toAnySupportedFormat(mesh, 'mesh.ply');
